dotnet/DDB.Bindings/DroneDB.cs

```diff
@@ -47,19 +62,39 @@ public static string Init(string directory)
 
         public static List<Entry> Add(string ddbPath, string path, bool recursive = false)
         {
+
+            if (string.IsNullOrWhiteSpace(path))
+                throw new DDBException("Path should not be null or empty");
+
             return Add(ddbPath, new[] { path }, recursive);
+
         }
         public static List<Entry> Add(string ddbPath, string[] paths, bool recursive = false)
         {
-            if (_Add(ddbPath, paths, paths.Length, out var output, recursive) != DDBError.DDBERR_NONE)
-                throw new DDBException(GetLastError());
 
-            var json = Marshal.PtrToStringAnsi(output);
+            if (string.IsNullOrWhiteSpace(ddbPath))
```

Comment:
Same :)

We do NOT want to add any logic inside the bindings (as much as its feasible anyway) that could then cause different behaviors between other language bindings / command line behavior.

Reply:
See https://github.com/DroneDB/DroneDB/pull/115#issuecomment-709561637